declarations/WebpackOptions.d.ts

```diff
@@ -756,6 +756,10 @@ export interface FileCacheOptions {
 	 * Locations for the cache (defaults to cacheDirectory / name).
 	 */
 	cacheLocation?: string;
+	/**
+	 * Write relative to cache folder paths instead of absolute
+	 */
+	contextify?: boolean;
```

Comment:
Maybe we should avoid this option and always write relative paths for better 0cjs and DX

Reply:
I followed @sokra suggestions in https://github.com/webpack/webpack/issues/10400 . I have set `contextify` to true by default